extensions/amp-pinterest/0.1/test/test-amp-pinterest.js

```diff
@@ -16,15 +16,19 @@
 
 import '../amp-pinterest';
 
-
 describes.realWin('amp-pinterest', {
   amp: {
     runtimeOn: false,
     ampdoc: 'single',
+    mockFetch: true,
     extensions: ['amp-pinterest'],
   },
 }, env => {
 
+  function widgetURL(pinID) {
+    return '^https://widgets.pinterest.com/v3/pidgets/pins/info/?pin_ids=' + pinID;
```

Comment:
Noticed there was a Url mismatch. AMP always append an extra QA param during fetchm so instead using a `^` here which tells the mock to just do a `startsWith` rather than a full equality.

Reply:
Ack